packages/awslint/lib/rules/docs.ts

```diff
@@ -71,6 +72,21 @@ docsLinter.add({
   },
 });
 
+docsLinter.add({
+  code: 'no-experimental-apis',
+  message: 'The use of @experimental in not allowed (either directly or via parent class)',
```

Comment:
> (either directly or via parent class)

what's this mean?

Reply:
I removed it, I agree it does not add value to the error. What I was referring to is the fact that if a parent class will add a @experimental annotation, the error will be thrown for every child class as well as the parent, regardless of whether the inherited property have the annotation. But since we don't allow the annotation at all, it will not be on the parent class.